Google.Api.Gax.Rest/ClientBuilderBase.cs

```diff
@@ -131,13 +141,16 @@ protected virtual IConfigurableHttpClientInitializer GetHttpClientInitializer()
                 CredentialsPath != null ? GoogleCredential.FromFile(CredentialsPath) :
                 JsonCredentials != null ? GoogleCredential.FromJson(JsonCredentials) :
                 null; // Use default credentials (maybe - see below)
-            // While we accept any credentials that are specified even if there's an API key,
-            // we don't try to load default credentials.
+                      // While we accept any credentials that are specified even if there's an API key,
+                      // we don't try to load default credentials.
```

Comment:
Revert these lines - this isn't part of the comment on the line above; it's a comment for the lines below.

Reply:
(Maybe add a blank line above to clarify that?)

Reply:
Done, added blank line.